.github/workflows/chart-test.yaml

```diff
@@ -130,26 +130,38 @@ jobs:
 
       - name: Create kind cluster
         uses: helm/kind-action@v1.10.0
-        if: ${{ steps.list-changed.outputs.changed == 'true' || github.ref == 'refs/heads/main' }}
 
       - name: Install SealedSecrets Helm Chart
-        if: ${{ steps.list-changed.outputs.changed == 'true' || github.ref == 'refs/heads/main' }}
         run: |
           helm repo add sealed-secrets https://bitnami-labs.github.io/sealed-secrets
           helm install sealed-secrets sealed-secrets/sealed-secrets
 
+      - name: Create posit-test namespace
+        run: kubectl create namespace posit-test
+
+      - name: Create License File Secrets
+        run: |
+          echo "${{ secrets.PWB_LICENSE_FILE }}" > pwb.lic
+          kubectl create secret generic pwb-license --from-file=pwb.lic --namespace posit-test
+          rm pwb.lic
+          echo "${{ secrets.RSC_LICENSE_FILE }}" > rsc.lic
```

Comment:
I think they refer to it as PCT (Posit Connect) now

Reply:
Thanks!

Reply:
😭 We are the Posit Percent? 🤮 @edavidaja 

It might still be "RSC", as hinted by this reference:
https://github.com/rstudio/helm/blob/b578ab99908643f3c6aa0d91b4b04fc0cb7d83e7/charts/rstudio-connect/templates/deployment.yaml#L107

Reply:
The license file I generated from https://connect.posit.it/support/license_management/ is named with `PCT` in it 

<img width="830" alt="image" src="https://github.com/user-attachments/assets/b45a661b-222d-45ae-acc4-43d3e33bbcaf">

PPM has PPM, and PWB has PWB.  🤷

Reply:
Yeah. PCT is being adopted by folks. The `envVarPrefix` still uses "RSC".